ng-eject missing @angular-devkit/core

### Versions
```
Angular CLI: 1.6.3
Node: 8.9.3
OS: win32 x64
Angular: 5.2.1
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

@angular/cli: 1.6.3
@angular-devkit/build-optimizer: 0.0.41
@angular-devkit/core: 0.0.28
@angular-devkit/schematics: 0.0.51
@ngtools/json-schema: 1.1.0
@ngtools/webpack: 1.9.3
@schematics/angular: 0.1.16
typescript: 2.4.2
webpack: 3.10.0
```

### Repro steps
* Step 1
ng new app-ejected

* Step 2
cd app-ejected

* Step 3
ng eject


### Observed behavior
```
module.js:538
    throw err;
    ^

Error: Cannot find module '@angular-devkit/core'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Dev\Lab\lib-lab\node_modules\@angular-devkit\schematics\src\tree\virtual.js:10:16)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
```

### Desired behavior
Be able to eject the project immediately after the creation, without any hassle 


### Mention any other details that might be useful (optional)

To make it working, I do need to run npm i -D @angular-devkit/core and then ng eject

--- a/packages/@angular/cli/tasks/eject.ts
+++ b/packages/@angular/cli/tasks/eject.ts
@@ -581,6 +581,8 @@ export default Task.extend({
 
         // Update all loaders from webpack, plus postcss plugins.
         [
+          '@angular-devkit/core',
+          '@ngtools/webpack',
           'webpack',
           'autoprefixer',
           'css-loader',

--- a/tests/e2e/utils/process.ts
+++ b/tests/e2e/utils/process.ts
@@ -178,8 +178,8 @@ export function ng(...args: string[]) {
         .then(() => {
           if (!npmInstalledEject) {
             npmInstalledEject = true;
-            // We need to run npm install on the first eject.
-            return silentNpm('install');
+            // We need to delete node_modules, then run npm install on the first eject.
+            return rimraf('node_modules').then(() => silentNpm('install'));
           }
         })
         .then(() => rimraf('dist'))

